Time taken to run the queries on ./repositories/Django/thread0/0xAliRaza_habitly/0xAliRaza-habitly-4dabf57: 634.525 seconds.
Analysis failed for: ./repositories/Django/thread0/20tab_django-continuous-delivery/20tab-django-continuous-delivery-f5db38d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/20tab_django-continuous-delivery/20tab-django-continuous-delivery-f5db38d -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/20tab_django-continuous-delivery/20tab-django-continuous-delivery-f5db38d: 150.587 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Abhishek-Gawade-programmer_socialfly-instagram-clone/Abhishek-Gawade-programmer-socialfly-instagram-clone-8b16773: 602.672 seconds.
Analysis failed for: ./repositories/Django/thread0/Anitaachu_Django-E-commerce-RESTAPI/Anitaachu-Django-E-commerce-RESTAPI-7514e11 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Anitaachu_Django-E-commerce-RESTAPI/Anitaachu-Django-E-commerce-RESTAPI-7514e11 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Anitaachu_Django-E-commerce-RESTAPI/Anitaachu-Django-E-commerce-RESTAPI-7514e11: 141.684 seconds.
Analysis failed for: ./repositories/Django/thread0/AnuragTimilsina_Tourism_app/AnuragTimilsina-Tourism_app-46f1710 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/AnuragTimilsina_Tourism_app/AnuragTimilsina-Tourism_app-46f1710 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/AnuragTimilsina_Tourism_app/AnuragTimilsina-Tourism_app-46f1710: 141.677 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ArnaudLevaufre_Cinema/ArnaudLevaufre-Cinema-cbe55bf: 407.256 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Asheshshrestha_SmartWasteManagement/Asheshshrestha-SmartWasteManagement-b0fd066: 342.511 seconds.
Analysis failed for: ./repositories/Django/thread0/Azure-Samples_azure-django-postgres-flexible-aca/Azure-Samples-azure-django-postgres-flexible-aca-2b6de4d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Azure-Samples_azure-django-postgres-flexible-aca/Azure-Samples-azure-django-postgres-flexible-aca-2b6de4d -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Azure-Samples_azure-django-postgres-flexible-aca/Azure-Samples-azure-django-postgres-flexible-aca-2b6de4d: 73.058 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Bulkmailer_BulkMailer-Backend/Bulkmailer-BulkMailer-Backend-8bc5216: 349.232 seconds.
Time taken to run the queries on ./repositories/Django/thread0/CambridgeSemiticsLab_nena/CambridgeSemiticsLab-nena-b297a93: 334.199 seconds.
Time taken to run the queries on ./repositories/Django/thread0/CaptainDynamight_SAMS/CaptainDynamight-SAMS-5608c62: 436.77 seconds.
Analysis failed for: ./repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2 Reason: Error: Command failed: codeql database create ./repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2-database --language=python --source-root ./repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread0/Chudopal_my_notes/Chudopal-my_notes-eb94dd2: 20.589 seconds.
Analysis failed for: ./repositories/Django/thread0/Code-Institute-Submissions_django-ecommerce-1/Code-Institute-Submissions-django-ecommerce-1-2481275 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Code-Institute-Submissions_django-ecommerce-1/Code-Institute-Submissions-django-ecommerce-1-2481275 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Code-Institute-Submissions_django-ecommerce-1/Code-Institute-Submissions-django-ecommerce-1-2481275: 111.62 seconds.
Analysis failed for: ./repositories/Django/thread0/CodeTheChangeUBC_reBOOT/CodeTheChangeUBC-reBOOT-d2abb22 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/CodeTheChangeUBC_reBOOT/CodeTheChangeUBC-reBOOT-d2abb22 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/CodeTheChangeUBC_reBOOT/CodeTheChangeUBC-reBOOT-d2abb22: 112.942 seconds.
Time taken to run the queries on ./repositories/Django/thread0/CodeYellowBV_django-binder/CodeYellowBV-django-binder-9e8bd20: 548.49 seconds.
Analysis failed for: ./repositories/Django/thread0/DakaraProject_dakara-server/DakaraProject-dakara-server-baf117e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/DakaraProject_dakara-server/DakaraProject-dakara-server-baf117e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/DakaraProject_dakara-server/DakaraProject-dakara-server-baf117e: 121.867 seconds.
Time taken to run the queries on ./repositories/Django/thread0/E-ARK-Software_earkweb/E-ARK-Software-earkweb-b1f9034: 588.892 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ESGF_esgf-compute-wps/ESGF-esgf-compute-wps-82fb3e7: 760.493 seconds.
Time taken to run the queries on ./repositories/Django/thread0/FireCARES_firecares/FireCARES-firecares-aa708d4: 652.472 seconds.
Analysis failed for: ./repositories/Django/thread0/FixMyBerlin_fixmy.platform/FixMyBerlin-fixmy.platform-c872880 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/FixMyBerlin_fixmy.platform/FixMyBerlin-fixmy.platform-c872880 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/FixMyBerlin_fixmy.platform/FixMyBerlin-fixmy.platform-c872880: 126.414 seconds.
Analysis failed for: ./repositories/Django/thread0/Frojd_Wagtail-Pipit/Frojd-Wagtail-Pipit-f2b9326 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Frojd_Wagtail-Pipit/Frojd-Wagtail-Pipit-f2b9326 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Frojd_Wagtail-Pipit/Frojd-Wagtail-Pipit-f2b9326: 87.183 seconds.
Time taken to run the queries on ./repositories/Django/thread0/GoogleCloudPlatform_hpc-toolkit/GoogleCloudPlatform-hpc-toolkit-75a04d4: 574.413 seconds.
Time taken to run the queries on ./repositories/Django/thread0/HumanSignal_label-studio/HumanSignal-label-studio-4acc1bf: 177.014 seconds.
Time taken to run the queries on ./repositories/Django/thread0/JSv4_OpenContracts/JSv4-OpenContracts-be3de1c: 547.487 seconds.
Analysis failed for: ./repositories/Django/thread0/Jauharmuhammed_social-network-django-server/Jauharmuhammed-social-network-django-server-559174d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Jauharmuhammed_social-network-django-server/Jauharmuhammed-social-network-django-server-559174d -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Jauharmuhammed_social-network-django-server/Jauharmuhammed-social-network-django-server-559174d: 205.465 seconds.
Time taken to run the queries on ./repositories/Django/thread0/JayMalde_facial-recognition-system/JayMalde-facial-recognition-system-cbc80f8: 1001.533 seconds.
Time taken to run the queries on ./repositories/Django/thread0/KiOui_mentorpunten/KiOui-mentorpunten-4307f7f: 779.034 seconds.
Time taken to run the queries on ./repositories/Django/thread0/LCOGT_neoexchange/LCOGT-neoexchange-7b21f32: 1117.772 seconds.
Analysis failed for: ./repositories/Django/thread0/MSU-AI_Attendance-Project/MSU-AI-Attendance-Project-0921af6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/MSU-AI_Attendance-Project/MSU-AI-Attendance-Project-0921af6 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/MSU-AI_Attendance-Project/MSU-AI-Attendance-Project-0921af6: 225.115 seconds.
Analysis failed for: ./repositories/Django/thread0/MakeMonmouth_mventory/MakeMonmouth-mventory-80b1732 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/MakeMonmouth_mventory/MakeMonmouth-mventory-80b1732 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/MakeMonmouth_mventory/MakeMonmouth-mventory-80b1732: 177.702 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Mohitsaini2028_Shocally/Mohitsaini2028-Shocally-cbd2552: 973.326 seconds.
Time taken to run the queries on ./repositories/Django/thread0/MonAPI-xyz_MonAPI/MonAPI-xyz-MonAPI-efeefe7: 773.143 seconds.
Time taken to run the queries on ./repositories/Django/thread0/MozillaSecurity_FuzzManager/MozillaSecurity-FuzzManager-990a0e1: 901.162 seconds.
Time taken to run the queries on ./repositories/Django/thread0/MrDoomBringer_TidyPlan/MrDoomBringer-TidyPlan-2b2642f: 646.611 seconds.
Analysis failed for: ./repositories/Django/thread0/Nezhinskiy_Assets-Loop/Nezhinskiy-Assets-Loop-d35740b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Nezhinskiy_Assets-Loop/Nezhinskiy-Assets-Loop-d35740b -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Nezhinskiy_Assets-Loop/Nezhinskiy-Assets-Loop-d35740b: 177.34 seconds.
Analysis failed for: ./repositories/Django/thread0/NoahFinberg_djangostarterproject/NoahFinberg-djangostarterproject-e56c018 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/NoahFinberg_djangostarterproject/NoahFinberg-djangostarterproject-e56c018 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/NoahFinberg_djangostarterproject/NoahFinberg-djangostarterproject-e56c018: 139.284 seconds.
Time taken to run the queries on ./repositories/Django/thread0/OpenHumans_quantified-flu/OpenHumans-quantified-flu-3ee9bde: 735.263 seconds.
Analysis failed for: ./repositories/Django/thread0/OpenUpSA_wazimap-ng/OpenUpSA-wazimap-ng-c1f33ae Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/OpenUpSA_wazimap-ng/OpenUpSA-wazimap-ng-c1f33ae -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/OpenUpSA_wazimap-ng/OpenUpSA-wazimap-ng-c1f33ae: 231.086 seconds.
Time taken to run the queries on ./repositories/Django/thread0/SCPru_RuFoundation/SCPru-RuFoundation-2af562d: 640.569 seconds.
Time taken to run the queries on ./repositories/Django/thread0/SJVAir_sjvair.com/SJVAir-sjvair.com-45f4fbd: 829.92 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Sahilamin219_Django_Project/Sahilamin219-Django_Project-d386e4b: 745.899 seconds.
Analysis failed for: ./repositories/Django/thread0/ScienceCommons_curate_science/ScienceCommons-curate_science-4e4072e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ScienceCommons_curate_science/ScienceCommons-curate_science-4e4072e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ScienceCommons_curate_science/ScienceCommons-curate_science-4e4072e: 151.354 seconds.
Analysis failed for: ./repositories/Django/thread0/ShahandFahad_Meta-Back-End-Developer/ShahandFahad-Meta-Back-End-Developer-493f609 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ShahandFahad_Meta-Back-End-Developer/ShahandFahad-Meta-Back-End-Developer-493f609 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ShahandFahad_Meta-Back-End-Developer/ShahandFahad-Meta-Back-End-Developer-493f609: 168.544 seconds.
Time taken to run the queries on ./repositories/Django/thread0/TaeBbong_drf_for_backend/TaeBbong-drf_for_backend-fab66f9: 638.802 seconds.
Time taken to run the queries on ./repositories/Django/thread0/Team-uMigrate_umigrate/Team-uMigrate-umigrate-b1125ad: 752.706 seconds.
Time taken to run the queries on ./repositories/Django/thread0/TobySuch_Shifter/TobySuch-Shifter-2cf9305: 536.759 seconds.
Analysis failed for: ./repositories/Django/thread0/Twopersonwork_LinkedIn-Clone-Django/Twopersonwork-LinkedIn-Clone-Django-a39d210 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/Twopersonwork_LinkedIn-Clone-Django/Twopersonwork-LinkedIn-Clone-Django-a39d210 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/Twopersonwork_LinkedIn-Clone-Django/Twopersonwork-LinkedIn-Clone-Django-a39d210: 172.038 seconds.
Analysis failed for: ./repositories/Django/thread0/UplandsDynamic_simple-stock-management/UplandsDynamic-simple-stock-management-683461b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/UplandsDynamic_simple-stock-management/UplandsDynamic-simple-stock-management-683461b -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/UplandsDynamic_simple-stock-management/UplandsDynamic-simple-stock-management-683461b: 139.976 seconds.
Time taken to run the queries on ./repositories/Django/thread0/YuZaGa_E-Vote-Django/YuZaGa-E-Vote-Django-638a6d1: 531.359 seconds.
Time taken to run the queries on ./repositories/Django/thread0/YuZongYangHi_ansible-cmdb-platform/YuZongYangHi-ansible-cmdb-platform-91de5a8: 385.365 seconds.
Time taken to run the queries on ./repositories/Django/thread0/abhishek2x_VITBOOK/abhishek2x-VITBOOK-1038c40: 336.909 seconds.
Analysis failed for: ./repositories/Django/thread0/ablab_nerpa/ablab-nerpa-75b9cc6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ablab_nerpa/ablab-nerpa-75b9cc6 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ablab_nerpa/ablab-nerpa-75b9cc6: 126.425 seconds.
Analysis failed for: ./repositories/Django/thread0/activist-org_activist/activist-org-activist-9ef2b58 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/activist-org_activist/activist-org-activist-9ef2b58 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/activist-org_activist/activist-org-activist-9ef2b58: 77.34 seconds.
Analysis failed for: ./repositories/Django/thread0/albert0796_MachineLearning/albert0796-MachineLearning-3c4a9db Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/albert0796_MachineLearning/albert0796-MachineLearning-3c4a9db -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/albert0796_MachineLearning/albert0796-MachineLearning-3c4a9db: 147.621 seconds.
Analysis failed for: ./repositories/Django/thread0/ankicommunity_anki-devops-services/ankicommunity-anki-devops-services-bde88cd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ankicommunity_anki-devops-services/ankicommunity-anki-devops-services-bde88cd -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ankicommunity_anki-devops-services/ankicommunity-anki-devops-services-bde88cd: 76.986 seconds.
Analysis failed for: ./repositories/Django/thread0/arunism_Instagram-Clone/arunism-Instagram-Clone-787935e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/arunism_Instagram-Clone/arunism-Instagram-Clone-787935e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/arunism_Instagram-Clone/arunism-Instagram-Clone-787935e: 85.112 seconds.
Time taken to run the queries on ./repositories/Django/thread0/asifo1_bug-binder/as-shahriar-bug-binder-8bf7d8b: 353.31 seconds.
Time taken to run the queries on ./repositories/Django/thread0/autodo-app_autodo/autodo-app-autodo-33bc395: 324.7 seconds.
Time taken to run the queries on ./repositories/Django/thread0/billvsme_vmaig_blog/billvsme-vmaig_blog-5494915: 360.029 seconds.
Time taken to run the queries on ./repositories/Django/thread0/biocodellc_localcontexts_db/biocodellc-localcontexts_db-3d0186b: 425.294 seconds.
Analysis failed for: ./repositories/Django/thread0/bluemoo_django-with-microk8s/bluemoo-django-with-microk8s-862ba19 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/bluemoo_django-with-microk8s/bluemoo-django-with-microk8s-862ba19 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/bluemoo_django-with-microk8s/bluemoo-django-with-microk8s-862ba19: 67.7 seconds.
Time taken to run the queries on ./repositories/Django/thread0/bmun_huxley/bmun-huxley-271aeac: 428.243 seconds.
Time taken to run the queries on ./repositories/Django/thread0/bookwyrm-social_bookwyrm/bookwyrm-social-bookwyrm-1fabe51: 509.678 seconds.
Analysis failed for: ./repositories/Django/thread0/buildlyio_buildly-core/buildlyio-buildly-core-93e3b26 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/buildlyio_buildly-core/buildlyio-buildly-core-93e3b26 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/buildlyio_buildly-core/buildlyio-buildly-core-93e3b26: 100.533 seconds.
Analysis failed for: ./repositories/Django/thread0/bytedeck_bytedeck/bytedeck-bytedeck-d7948bc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/bytedeck_bytedeck/bytedeck-bytedeck-d7948bc -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/bytedeck_bytedeck/bytedeck-bytedeck-d7948bc: 158.643 seconds.
Time taken to run the queries on ./repositories/Django/thread0/cfedermann_Appraise/cfedermann-Appraise-2cce477: 321.633 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ckc-org_skeletor/ckc-org-skeletor-ffe5106: 316.197 seconds.
Analysis failed for: ./repositories/Django/thread0/code4romania_seismic-risc/code4romania-seismic-risc-56720ed Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/code4romania_seismic-risc/code4romania-seismic-risc-56720ed -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/code4romania_seismic-risc/code4romania-seismic-risc-56720ed: 82.952 seconds.
Analysis failed for: ./repositories/Django/thread0/dabapps_django-s3-file-upload-server/dabapps-django-s3-file-upload-server-42eb936 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/dabapps_django-s3-file-upload-server/dabapps-django-s3-file-upload-server-42eb936 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/dabapps_django-s3-file-upload-server/dabapps-django-s3-file-upload-server-42eb936: 82.991 seconds.
Analysis failed for: ./repositories/Django/thread0/danidee10_Chatire/danidee10-Chatire-8e436d4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/danidee10_Chatire/danidee10-Chatire-8e436d4 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/danidee10_Chatire/danidee10-Chatire-8e436d4: 95.67 seconds.
Analysis failed for: ./repositories/Django/thread0/datarobot-community_ai_engineering/datarobot-community-ai_engineering-abf4883 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/datarobot-community_ai_engineering/datarobot-community-ai_engineering-abf4883 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/datarobot-community_ai_engineering/datarobot-community-ai_engineering-abf4883: 112.982 seconds.
Analysis failed for: ./repositories/Django/thread0/datosgobar_series-tiempo-ar-api/datosgobar-series-tiempo-ar-api-6b553c5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/datosgobar_series-tiempo-ar-api/datosgobar-series-tiempo-ar-api-6b553c5 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/datosgobar_series-tiempo-ar-api/datosgobar-series-tiempo-ar-api-6b553c5: 142.729 seconds.
Analysis failed for: ./repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766 Reason: Error: Command failed: codeql database create ./repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766-database --language=python --source-root ./repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread0/dearxuany_Sharon_Technology_learning_note/dearxuany-Sharon_Technology_learning_note-6fb8766: 9.462 seconds.
Analysis failed for: ./repositories/Django/thread0/dev0088_django-oauth-react-admin-dashboard/dev0088-django-oauth-react-admin-dashboard-fbae268 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/dev0088_django-oauth-react-admin-dashboard/dev0088-django-oauth-react-admin-dashboard-fbae268 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/dev0088_django-oauth-react-admin-dashboard/dev0088-django-oauth-react-admin-dashboard-fbae268: 92.352 seconds.
Time taken to run the queries on ./repositories/Django/thread0/discolabs_django-shopify-auth/discolabs-django-shopify-auth-73fb6bf: 410.686 seconds.
Analysis failed for: ./repositories/Django/thread0/drbeni_malquarium/drbeni-malquarium-42a87bd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/drbeni_malquarium/drbeni-malquarium-42a87bd -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/drbeni_malquarium/drbeni-malquarium-42a87bd: 106.293 seconds.
Time taken to run the queries on ./repositories/Django/thread0/drshahizan_special-topic-data-engineering/drshahizan-special-topic-data-engineering-1d02811: 518.155 seconds.
Analysis failed for: ./repositories/Django/thread0/eastend-street_jotdown/eastend-street-jotdown-b80175f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/eastend-street_jotdown/eastend-street-jotdown-b80175f -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/eastend-street_jotdown/eastend-street-jotdown-b80175f: 85.972 seconds.
Time taken to run the queries on ./repositories/Django/thread0/electionscience_vote.electionscience.org/electionscience-vote.electionscience.org-611e363: 345.568 seconds.
Time taken to run the queries on ./repositories/Django/thread0/emaula_site/emaula-site-8e56338: 327.624 seconds.
Analysis failed for: ./repositories/Django/thread0/epicserve_django-base-site/epicserve-django-base-site-d0ed5c0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/epicserve_django-base-site/epicserve-django-base-site-d0ed5c0 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/epicserve_django-base-site/epicserve-django-base-site-d0ed5c0: 81.359 seconds.
Analysis failed for: ./repositories/Django/thread0/fabiocaccamo_django-maintenance-mode/fabiocaccamo-django-maintenance-mode-70b9dcd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/fabiocaccamo_django-maintenance-mode/fabiocaccamo-django-maintenance-mode-70b9dcd -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/fabiocaccamo_django-maintenance-mode/fabiocaccamo-django-maintenance-mode-70b9dcd: 88.337 seconds.
Time taken to run the queries on ./repositories/Django/thread0/fayomihorace_django-netflix-clone/fayomihorace-django-netflix-clone-23db7a4: 336.631 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ffAudio_LicenseServer/ffAudio-LicenseServer-0c26d50: 345.242 seconds.
Analysis failed for: ./repositories/Django/thread0/fortunto2_bashair/fortunto2-bashair-3f39fcd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/fortunto2_bashair/fortunto2-bashair-3f39fcd -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/fortunto2_bashair/fortunto2-bashair-3f39fcd: 113.923 seconds.
Time taken to run the queries on ./repositories/Django/thread0/freedombox_FreedomBox/freedombox-FreedomBox-a9b77bb: 517.482 seconds.
Analysis failed for: ./repositories/Django/thread0/fussionlab_VueJs-Django/fussionlab-VueJs-Django-a6c4425 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/fussionlab_VueJs-Django/fussionlab-VueJs-Django-a6c4425 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/fussionlab_VueJs-Django/fussionlab-VueJs-Django-a6c4425: 84.37 seconds.
Time taken to run the queries on ./repositories/Django/thread0/geetnsh2k1_B-Bhukkad-Food-Delivery-Web-Application/geetnsh2k1-B-Bhukkad-Food-Delivery-Web-Application-b405369: 324.031 seconds.
Analysis failed for: ./repositories/Django/thread0/geropellicer_DjangoReact/geropellicer-DjangoReact-ef62b12 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/geropellicer_DjangoReact/geropellicer-DjangoReact-ef62b12 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/geropellicer_DjangoReact/geropellicer-DjangoReact-ef62b12: 93.101 seconds.
Time taken to run the queries on ./repositories/Django/thread0/getsentry_sentry-python/getsentry-sentry-python-2eeb8c5: 510.406 seconds.
Time taken to run the queries on ./repositories/Django/thread0/gklyne_annalist/gklyne-annalist-4e17e51: 515.143 seconds.
Time taken to run the queries on ./repositories/Django/thread0/go-recordkeeper_go-recordkeeper/go-recordkeeper-go-recordkeeper-21f45df: 374.278 seconds.
Analysis failed for: ./repositories/Django/thread0/hadijahkyampeire_taxi-app-Django-React/hadijahkyampeire-taxi-app-Django-React-804a12e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/hadijahkyampeire_taxi-app-Django-React/hadijahkyampeire-taxi-app-Django-React-804a12e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/hadijahkyampeire_taxi-app-Django-React/hadijahkyampeire-taxi-app-Django-React-804a12e: 79.104 seconds.
Analysis failed for: ./repositories/Django/thread0/holoviz_panel/holoviz-panel-d6c6814 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/holoviz_panel/holoviz-panel-d6c6814 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/holoviz_panel/holoviz-panel-d6c6814: 164.854 seconds.
Analysis failed for: ./repositories/Django/thread0/hualuoo_dormitory_menage_system/hualuoo-dormitory_menage_system-2c82c89 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/hualuoo_dormitory_menage_system/hualuoo-dormitory_menage_system-2c82c89 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/hualuoo_dormitory_menage_system/hualuoo-dormitory_menage_system-2c82c89: 109.85 seconds.
Analysis failed for: ./repositories/Django/thread0/hygull_sbadmin/hygull-sbadmin-d58e4e1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/hygull_sbadmin/hygull-sbadmin-d58e4e1 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/hygull_sbadmin/hygull-sbadmin-d58e4e1: 104.255 seconds.
Analysis failed for: ./repositories/Django/thread0/ionata_ionata-django-skeleton/ionata-ionata-django-skeleton-8aace15 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ionata_ionata-django-skeleton/ionata-ionata-django-skeleton-8aace15 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ionata_ionata-django-skeleton/ionata-ionata-django-skeleton-8aace15: 82.04 seconds.
Analysis failed for: ./repositories/Django/thread0/ispc-programador2022_GPQRR5/ispc-programador2022-GPQRR5-452a923 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ispc-programador2022_GPQRR5/ispc-programador2022-GPQRR5-452a923 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ispc-programador2022_GPQRR5/ispc-programador2022-GPQRR5-452a923: 126.572 seconds.
Time taken to run the queries on ./repositories/Django/thread0/italia_spid-cie-oidc-django/italia-spid-cie-oidc-django-8746747: 403.299 seconds.
Analysis failed for: ./repositories/Django/thread0/iu5team_iu5web-fall-2016/iu5team-iu5web-fall-2016-54bce3c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/iu5team_iu5web-fall-2016/iu5team-iu5web-fall-2016-54bce3c -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/iu5team_iu5web-fall-2016/iu5team-iu5web-fall-2016-54bce3c: 142.769 seconds.
Time taken to run the queries on ./repositories/Django/thread0/jedie_django-reversion-compare/jedie-django-reversion-compare-2ed5b3e: 356.325 seconds.
Analysis failed for: ./repositories/Django/thread0/jhanjhenyi_django-line-echobot/jhanjhenyi-django-line-echobot-50c11e2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/jhanjhenyi_django-line-echobot/jhanjhenyi-django-line-echobot-50c11e2 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/jhanjhenyi_django-line-echobot/jhanjhenyi-django-line-echobot-50c11e2: 110.427 seconds.
Analysis failed for: ./repositories/Django/thread0/jhuapl-boss_boss-oidc/jhuapl-boss-boss-oidc-738afa2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/jhuapl-boss_boss-oidc/jhuapl-boss-boss-oidc-738afa2 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/jhuapl-boss_boss-oidc/jhuapl-boss-boss-oidc-738afa2: 108.52 seconds.
Analysis failed for: ./repositories/Django/thread0/keithamoss_demsausage/keithamoss-demsausage-8d5ebec Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/keithamoss_demsausage/keithamoss-demsausage-8d5ebec -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/keithamoss_demsausage/keithamoss-demsausage-8d5ebec: 98.815 seconds.
Time taken to run the queries on ./repositories/Django/thread0/koduj-z-klasa_python101/koduj-z-klasa-python101-accfca2: 525.389 seconds.
Time taken to run the queries on ./repositories/Django/thread0/koguz_TaskPointSystem/koguz-TaskPointSystem-1996f57: 348.956 seconds.
Analysis failed for: ./repositories/Django/thread0/kokimame_django-pdf-kanji/kokimame-django-pdf-kanji-2444ba8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/kokimame_django-pdf-kanji/kokimame-django-pdf-kanji-2444ba8 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/kokimame_django-pdf-kanji/kokimame-django-pdf-kanji-2444ba8: 93.677 seconds.
Analysis failed for: ./repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53 Reason: Error: Command failed: codeql database create ./repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53-database --language=python --source-root ./repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread0/letgodchan0_TIL/letgodchan0-TIL-e2c5e53: 14.223 seconds.
Analysis failed for: ./repositories/Django/thread0/lhfei_cloud-doc/lhfei-cloud-doc-32780c1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/lhfei_cloud-doc/lhfei-cloud-doc-32780c1 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/lhfei_cloud-doc/lhfei-cloud-doc-32780c1: 84.134 seconds.
Analysis failed for: ./repositories/Django/thread0/liaogx_imooc/liaogx-imooc-05407d1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/liaogx_imooc/liaogx-imooc-05407d1 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/liaogx_imooc/liaogx-imooc-05407d1: 85.74 seconds.
Time taken to run the queries on ./repositories/Django/thread0/mcdougallab_modeldb/mcdougallab-modeldb-0a2b7b6: 420.691 seconds.
Analysis failed for: ./repositories/Django/thread0/mdrhmn_react-dj-todoapp/mdrhmn-react-dj-todoapp-ff8c4bf Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/mdrhmn_react-dj-todoapp/mdrhmn-react-dj-todoapp-ff8c4bf -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/mdrhmn_react-dj-todoapp/mdrhmn-react-dj-todoapp-ff8c4bf: 84.078 seconds.
Time taken to run the queries on ./repositories/Django/thread0/membermatters_MemberMatters/membermatters-MemberMatters-04a8d4f: 423.225 seconds.
Analysis failed for: ./repositories/Django/thread0/mschwager_route-detect/mschwager-route-detect-ba92a57 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/mschwager_route-detect/mschwager-route-detect-ba92a57 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/mschwager_route-detect/mschwager-route-detect-ba92a57: 81.202 seconds.
Time taken to run the queries on ./repositories/Django/thread0/nature1995_Vue-Django-Shop-Website/nature1995-Vue-Django-Shop-Website-80d7317: 461.929 seconds.
Time taken to run the queries on ./repositories/Django/thread0/newfs_gobotany-app/newfs-gobotany-app-9030d08: 456.913 seconds.
Analysis failed for: ./repositories/Django/thread0/openwisp_docker-openwisp/openwisp-docker-openwisp-efbda73 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/openwisp_docker-openwisp/openwisp-docker-openwisp-efbda73 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/openwisp_docker-openwisp/openwisp-docker-openwisp-efbda73: 79.46 seconds.
Time taken to run the queries on ./repositories/Django/thread0/penn-state-dance-marathon_python3-saml-django/penn-state-dance-marathon-python3-saml-django-61c2712: 329.194 seconds.
Analysis failed for: ./repositories/Django/thread0/pik-software_django-service-boilerplate/pik-software-django-service-boilerplate-c39e221 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/pik-software_django-service-boilerplate/pik-software-django-service-boilerplate-c39e221 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/pik-software_django-service-boilerplate/pik-software-django-service-boilerplate-c39e221: 104.274 seconds.
Analysis failed for: ./repositories/Django/thread0/pjlantz_Hale/pjlantz-Hale-c3cda11 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/pjlantz_Hale/pjlantz-Hale-c3cda11 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/pjlantz_Hale/pjlantz-Hale-c3cda11: 106.037 seconds.
Time taken to run the queries on ./repositories/Django/thread0/project-icp_bee-pollinator-app/project-icp-bee-pollinator-app-9357755: 519.545 seconds.
Analysis failed for: ./repositories/Django/thread0/pulp_pulp-oci-images/pulp-pulp-oci-images-2556f83 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/pulp_pulp-oci-images/pulp-pulp-oci-images-2556f83 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/pulp_pulp-oci-images/pulp-pulp-oci-images-2556f83: 100.554 seconds.
Time taken to run the queries on ./repositories/Django/thread0/pyepye_django-magiclink/pyepye-django-magiclink-70a3ea9: 351.32 seconds.
Time taken to run the queries on ./repositories/Django/thread0/qatrackplus_qatrackplus/qatrackplus-qatrackplus-306592f: 736.322 seconds.
Analysis failed for: ./repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8 Reason: Error: Command failed: codeql database create ./repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8-database --language=python --source-root ./repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread0/ramwin_django-reference/ramwin-django-reference-c4ebdc8: 10.499 seconds.
Analysis failed for: ./repositories/Django/thread0/raychorn_dpmsf-microservice-framework/raychorn-dpmsf-microservice-framework-ed0753d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/raychorn_dpmsf-microservice-framework/raychorn-dpmsf-microservice-framework-ed0753d -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/raychorn_dpmsf-microservice-framework/raychorn-dpmsf-microservice-framework-ed0753d: 85.27 seconds.
Analysis failed for: ./repositories/Django/thread0/rg3915_comissao/rg3915-comissao-379da7d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/rg3915_comissao/rg3915-comissao-379da7d -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/rg3915_comissao/rg3915-comissao-379da7d: 89.862 seconds.
Analysis failed for: ./repositories/Django/thread0/rudranil723_mini-main/rudranil723-mini-main-5bf4642 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/rudranil723_mini-main/rudranil723-mini-main-5bf4642 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/rudranil723_mini-main/rudranil723-mini-main-5bf4642: 93.33 seconds.
Analysis failed for: ./repositories/Django/thread0/ruzhova_foodgram-project-react/ruzhova-foodgram-project-react-b0137c9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ruzhova_foodgram-project-react/ruzhova-foodgram-project-react-b0137c9 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ruzhova_foodgram-project-react/ruzhova-foodgram-project-react-b0137c9: 76.767 seconds.
Analysis failed for: ./repositories/Django/thread0/sajib1066_django-blog/sajib1066-django-blog-6f51c87 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/sajib1066_django-blog/sajib1066-django-blog-6f51c87 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/sajib1066_django-blog/sajib1066-django-blog-6f51c87: 90.235 seconds.
Time taken to run the queries on ./repositories/Django/thread0/shiep18_EIS2020/shiep18-EIS2020-92aa271: 3337.569 seconds.
Time taken to run the queries on ./repositories/Django/thread0/shuigedeng_taotao-cloud-project/shuigedeng-taotao-cloud-project-53a0d01: 2813.649 seconds.
Analysis failed for: ./repositories/Django/thread0/sixfeetup_cookiecutter-sixiedjango/sixfeetup-scaf-b1e8c38 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/sixfeetup_cookiecutter-sixiedjango/sixfeetup-scaf-b1e8c38 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/sixfeetup_cookiecutter-sixiedjango/sixfeetup-scaf-b1e8c38: 67.996 seconds.
Time taken to run the queries on ./repositories/Django/thread0/skkuding_skku-coding-platform/skkuding-skku-coding-platform-ffc2aec: 372.018 seconds.
Time taken to run the queries on ./repositories/Django/thread0/skynettw_dj4book/skynettw-dj4book-030edbd: 476.672 seconds.
Analysis failed for: ./repositories/Django/thread0/song-hee-1_customer-investment-service/song-hee-1-customer-investment-service-451af2d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/song-hee-1_customer-investment-service/song-hee-1-customer-investment-service-451af2d -l=python -t=11 -ct=0
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/song-hee-1_customer-investment-service/song-hee-1-customer-investment-service-451af2d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 5.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/song-hee-1_customer-investment-service/song-hee-1-customer-investment-service-451af2d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/song-hee-1_customer-investment-service/song-hee-1-customer-investment-service-451af2d: 210.607 seconds.
Analysis failed for: ./repositories/Django/thread0/st4lk_django-rest-social-auth/st4lk-django-rest-social-auth-fe2363b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/st4lk_django-rest-social-auth/st4lk-django-rest-social-auth-fe2363b -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/st4lk_django-rest-social-auth/st4lk-django-rest-social-auth-fe2363b: 96.039 seconds.
Analysis failed for: ./repositories/Django/thread0/subscriptionRescueTeam_save-my-youth/subscriptionRescueTeam-save-my-youth-2ef392e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/subscriptionRescueTeam_save-my-youth/subscriptionRescueTeam-save-my-youth-2ef392e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/subscriptionRescueTeam_save-my-youth/subscriptionRescueTeam-save-my-youth-2ef392e: 88.879 seconds.
Time taken to run the queries on ./repositories/Django/thread0/swsnu_swpp2021-team12/swsnu-swpp2021-team12-ff346c0: 284.882 seconds.
Time taken to run the queries on ./repositories/Django/thread0/swsnu_swppfall2022-team9/swsnu-swppfall2022-team9-0b4ec17: 281.048 seconds.
Analysis failed for: ./repositories/Django/thread0/thelabnyc_wagtail_blog/thelabnyc-wagtail_blog-71fe3ac Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/thelabnyc_wagtail_blog/thelabnyc-wagtail_blog-71fe3ac -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/thelabnyc_wagtail_blog/thelabnyc-wagtail_blog-71fe3ac: 76.857 seconds.
Time taken to run the queries on ./repositories/Django/thread0/thetruefuss_elmer/thetruefuss-elmer-753ec3b: 296.428 seconds.
Time taken to run the queries on ./repositories/Django/thread0/tubearchivist_tubearchivist/tubearchivist-tubearchivist-0ff27eb: 95.338 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ubccr_genesysv/ubccr-genesysv-d361de0: 275.665 seconds.
Analysis failed for: ./repositories/Django/thread0/ucdavis_ucdpuppet/ucdavis-ucdpuppet-9637737 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ucdavis_ucdpuppet/ucdavis-ucdpuppet-9637737 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ucdavis_ucdpuppet/ucdavis-ucdpuppet-9637737: 41.873 seconds.
Analysis failed for: ./repositories/Django/thread0/ude-soco_RIMA/ude-soco-RIMA-09b80a6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/ude-soco_RIMA/ude-soco-RIMA-09b80a6 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/ude-soco_RIMA/ude-soco-RIMA-09b80a6: 66.419 seconds.
Time taken to run the queries on ./repositories/Django/thread0/vinigracindo_sgce/vinigracindo-sgce-cf3d28f: 141.696 seconds.
Analysis failed for: ./repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2 Reason: Error: Command failed: codeql database create ./repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2-database --language=python --source-root ./repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread0/vpavlenko_web-2020/vpavlenko-web-2020-3bb54e2: 4.236 seconds.
Time taken to run the queries on ./repositories/Django/thread0/we1h0_SecurityManageFramwork/we1h0-SecurityManageFramwork-e706461: 159.585 seconds.
Time taken to run the queries on ./repositories/Django/thread0/welel_noted/welel-noted-0ec4017: 162.812 seconds.
Analysis failed for: ./repositories/Django/thread0/xinlei3166_xlcmdb/xinlei3166-xlcmdb-272f522 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/xinlei3166_xlcmdb/xinlei3166-xlcmdb-272f522 -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/xinlei3166_xlcmdb/xinlei3166-xlcmdb-272f522: 39.42 seconds.
Time taken to run the queries on ./repositories/Django/thread0/yeazin_Multi-Authors-advanced-Django-Blog/yeazin-Multi-Authors-advanced-Django-Blog-3799947: 138.241 seconds.
Time taken to run the queries on ./repositories/Django/thread0/yourlabs_ryzom/yourlabs-ryzom-425859e: 145.805 seconds.
Time taken to run the queries on ./repositories/Django/thread0/ysyesilyurt_Metu-CENG/ysyesilyurt-Metu-CENG-d6dde86: 175.421 seconds.
Analysis failed for: ./repositories/Django/thread0/yy-pku_yogaMaster/yy-pku-yogaMaster-4710a6b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/yy-pku_yogaMaster/yy-pku-yogaMaster-4710a6b -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/yy-pku_yogaMaster/yy-pku-yogaMaster-4710a6b: 44.313 seconds.
Time taken to run the queries on ./repositories/Django/thread0/zaxlct_imooc-django/zaxlct-imooc-django-daf1ced: 104.744 seconds.
Analysis failed for: ./repositories/Django/thread0/zhaochenyang20_Sino-Japanese-Relations-analysis/zhaochenyang20-Sino-Japanese-Relations-analysis-9da5c0f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/zhaochenyang20_Sino-Japanese-Relations-analysis/zhaochenyang20-Sino-Japanese-Relations-analysis-9da5c0f -l=python -t=11 -ct=0
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 30s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/zhaochenyang20_Sino-Japanese-Relations-analysis/zhaochenyang20-Sino-Japanese-Relations-analysis-9da5c0f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread0/zhaochenyang20_Sino-Japanese-Relations-analysis/zhaochenyang20-Sino-Japanese-Relations-analysis-9da5c0f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/zhaochenyang20_Sino-Japanese-Relations-analysis/zhaochenyang20-Sino-Japanese-Relations-analysis-9da5c0f: 94.691 seconds.
Analysis failed for: ./repositories/Django/thread0/zuri-training_auth_wiki_team_1_project/zuri-training-auth_wiki_team_1_project-439874e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread0/zuri-training_auth_wiki_team_1_project/zuri-training-auth_wiki_team_1_project-439874e -l=python -t=11 -ct=0
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread0/zuri-training_auth_wiki_team_1_project/zuri-training-auth_wiki_team_1_project-439874e: 17.221 seconds.
Time taken to run the queries and generate the statistics: 49685.687 seconds.
